calculus/derivative-rules.md

```diff
@@ -80,3 +80,58 @@ The results match! Some additional examples:
    f'(x) = \dd{}{f}{u} \dd{}{u}{x} = -2u^{-2} \cdot \dd{}{u}{x} =
      \frac{-4}{(1 + 2x)^2}
    \end{equation}
+
+```{example}
+Physics example: harmonic oscillator
```

Comment:
You have a build error preventing your preview from generating, but I think this will fix it:
```suggestion
```{example} Harmonic oscillator
```